include/rawpheno.backup.form.inc

```diff
@@ -165,7 +165,10 @@ function rawpheno_backup($form, &$form_state, $project_id = NULL, $command = 'de
         '#file_upload_max_size' => '10M',
         '#upload_location' => 'public://',
         '#upload_event' => 'manual',
-        '#upload_validators' => array('file_validate_extensions' => array('xlsx xls')),
+        '#upload_validators' => array(
+          'file_validate_extensions' => array('xlsx xls'),
+          'rawpheno_validate_spreadsheet_file' => array($form_state),
```

Comment:
This issue occurs when spreadsheet has an error. I appears to freeze in backup, while it won't let user to re-upload in upload page. After examining the code, I think custom validators are executed before form submit, thus errors show but the upload Drag and drop does not, whereas hook_file_validate() submits the form (reloads the dnd form) plus does the validation check, hence when error is present we see both error and the form.

Restored to using hook_file_validate().